interface/src/index.html

```diff
@@ -70,6 +70,10 @@
                 Download
               </a>
             </li>
+
+            <login-button></login-button>
+            <profile-button></profile-button>
```

Comment:
Agree as well. I think the way I would put it is that the button should "know" how to draw itself and what to do when clicked, but where (and if) the button appears should be up the the "parent" view.